master/buildbot/status/buildstep.py

```diff
@@ -397,7 +405,7 @@ def asDict(self, request=None):
                 for l in self.getLogs()]
 
         result["urls"] = self.getURLs()
-
+        result["urls"].update(self.getArtifactURLs())
```

Comment:
We shouldn't add artifacts urls to the 'urls', they should go to separate JSON element 'artifacts'